internal/testutil/constant/pem_ca.go

```diff
@@ -0,0 +1,27 @@
+package constant
```

Comment:
I would put `constant` in the same category as `helper` for a package name, consider if there's a better name, for example if this is stream related, `stream.DummyCert` may be a good one but feel free to consider any better option

Reply:
I have adjusted to make use of the embed feature, left the pem file under a testdata folder near the test were it is being used (https://stackoverflow.com/a/18431297 for reference)

Reply:
nice!  
for example at the beginning I was thinking about using it in CFN resources to include the JSON definition file and being able to validate required attributes, etc.